modules/uhttp/filterchain_builder.go

```diff
@@ -52,7 +53,15 @@ type filterChainBuilder struct {
 
 func defaultFilterChainBuilder(host service.Host) filterChainBuilder {
 	fcb := newFilterChainBuilder(host)
-	return fcb.AddFilters(contextFilter(host), panicFilter(host), tracingServerFilter(host), authorizationFilter(host))
+	scope := host.Metrics().Tagged(modules.HTTPTags)
+	return fcb.AddFilters(
+		panicFilter{scope.Counter("panic")},
+		metricsFilter{scope},
+		tracingServerFilter{scope},
+		authorizationFilter{
+			authClient:  host.AuthClient(),
+			authCounter: scope.Counter("auth.fail"),
```

Comment:
Okey. `tally.DefaultSeparator` is a "." It would be a pretty huge breaking change for tally to switch it out to something else. 
 I don't mind having to subScope everything, or keeping in one string, but IMAO having a "." allows readability to users.